client/dist/style.css

```diff
@@ -25,70 +25,226 @@
   grid-area: imageGallery;
 }
 
+.overViewContainer {
+  background-color:hotpink;
+  min-height: 75vh;
+  display: flex;
+}
+
 .productInfo {
-  background-color: blue;
   grid-area: productInfo;
+  display: flex;
+  background-color: blue;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  align-content: flex-start;
+  padding: .5rem;
+}
+
+.starReviewRow {
+display: flex;
+flex-direction: row;
+align-items: center;
+}
+
+.starOutline {
```

Comment:
Should some of the properties shared between all stars be moved, so we can share more of these properties?